src/types.ts

```diff
@@ -125,8 +125,8 @@ export interface TechnicalQuestion {
   questionId: string;
   title: string;
   created: string; // ISO 8601 date string
-  createdBy: Member;
-  approvedBy: Member;
+  createdBy: string;
```

Comment:
member id is a number iirc

Reply:
Yes but I'm returning it as a string of the actual name with the serializer.

Reply:
ohhhhhh I see